src/components/common/ConnectWallet/ConnectButton.tsx

```diff
@@ -12,10 +11,8 @@ export const ConnectButton = () => {
   }
 
   return (
-    <Box className={css.buttonContainer}>
-      <Button onClick={handleConnect} variant="contained" size="small" disableElevation>
-        Connect wallet
-      </Button>
-    </Box>
+    <Button onClick={handleConnect} variant="contained" size="small" disableElevation>
```

Comment:
I would suggest we reduce the horizontal padding on all buttons on mobile a bit. 48px in total seems a bit much

Reply:
Set it to 8px, except for the menu button, that one looks better with 16.